noir/noir_stdlib/src/bigint.nr

```diff
@@ -36,46 +36,321 @@ impl BigInt {
     #[builtin(bigint_from_le_bytes)]
     fn from_le_bytes(bytes: [u8], modulus: [u8]) -> BigInt {}
     #[builtin(bigint_to_le_bytes)]
-    pub fn to_le_bytes(self) -> [u8] {}
+    fn to_le_bytes(self) -> [u8] {}
 
-    pub fn bn254_fr_from_le_bytes(bytes: [u8]) -> BigInt {
-        BigInt::from_le_bytes(bytes, bn254_fr)
+    fn check_32_bytes(self: Self, other: BigInt) -> bool {
+        let bytes = self.to_le_bytes();
+        let o_bytes = other.to_le_bytes();
```

Comment:
Currently it is the case, though can we always guarantee that BigInt will always be 32bytes from Noirs perspective?

Reply:
Non, that's why the function explicitly has 32 in its name. It is a private function used by the 32 bits moduli to avoid code duplication.